### PR TITLE
Updated iot-edge on Zeus to v1.1.4

### PR DESCRIPTION
**Pull request by SimonSimCity**
Added a new bitbake-recipe for version 1.1.4 on Zeus

### CODE REVIEW

File: recipes-core/iotedge-daemon/iotedge-daemon.inc [hunk 1]
```diff
@@ -26,6 +26,7 @@ do_install () {
 
     # Data dir
     install -d -o iotedge -g iotedge "${D}${localstatedir}/lib/iotedge"
+    install -d -o iotedge -g iotedge "${D}${localstatedir}/log/iotedge"
```
Comment (darobs) [inline recipes-core/iotedge-daemon/iotedge-daemon.inc:29]: Duplicate line?

Comment (SimonSimCity) [reply recipes-core/iotedge-daemon/iotedge-daemon.inc:29]: Duplicate of where?

Comment (darobs) [reply recipes-core/iotedge-daemon/iotedge-daemon.inc:29]: missed the difference, nevermind!

**Review by darobs — APPROVED** [review]
👍

**Pull request merged**